pkg/pgmodel/ingestor/insert.go

```diff
@@ -381,6 +388,13 @@ func doInsert(conn pgxconn.PgxConn, reqs ...copyRequest) (err error) {
 	}
 	defer results.Close()
 
+	ts := maxtBatch.UnixNano()
+	ts /= 1000000 // Convert nano-secs to milli-secs.
```

Comment:
should we use the model.Time conversion functions here and elsewhere in the code?